workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Templates/UnityComponentUpdateManagerGenerator.tt

```diff
@@ -350,7 +350,7 @@ namespace <#= qualifiedNamespace #>
             {
                 if (!entityIdToAuthority.TryGetValue(entityId, out var authority))
                 {
-                    throw new ArgumentException("Entity not in view");
+                    throw new ArgumentException("Can not check authority; entity-component not in view.");
```

Comment:
can we add which entity id and component combination is failing?

Reply:
Yeah that would be useful. This could be spun out into a new exception with ids as fields so you don't have to string interpolate here

Reply:
Making a ticket for this. There are quite a few exceptions that need to be changed and I suspect that whatever type we make for this will need to change subtly in the next couple of weeks.